Theses.fr.js

```diff
@@ -0,0 +1,182 @@
+				"libraryCatalog": "www.theses.fr",
+				"thesisType": "thesis",
+				"university": "Lille 2",
+				"url": "http://www.theses.fr/2012LIL2S049",
+				"attachments": [
+					{
+						"title": "Snapshot"
+					}
+				],
+				"tags": [
+					{
+						"tag": "Stratégie de potentialisation Tuberculose Ethionamide Isoniazide Multi-résistance  en français"
```

Comment:
These tags look good, but need to be properly separated out to be useful. Is there no place the tags are listed more systematically?